src/libraries/System.IO.Hashing/tests/System.IO.Hashing.Tests.csproj

```diff
@@ -8,8 +8,8 @@
     <Compile Include="Crc64Tests.cs" />
     <Compile Include="NonCryptoHashBaseTests.cs" />
     <Compile Include="NonCryptoHashTestDriver.cs" />
+    <Compile Include="XxHash3Tests.cs" />
     <Compile Include="XxHash32Tests.cs" />
-    <Compile Include="XxHash32Tests.007.cs" />
```

Comment:
Since this file wasn't also deleted I assume this is an error.  (If not, then "why?")

Reply:
I have no idea what happened there.  I'll put it back.